src/index.ts

```diff
@@ -181,6 +210,24 @@ namespace Private {
   /**
    * A counter for unique IDs.
    */
-  export
   let id = 0;
+
+  /**
+   * A cache for the currently active LaTeX previews.
+   */
+  export
+  const previews = new Set<string>();
+
+  /**
+   * Create an error panel widget.
+   */
+  export
+  function createErrorPanel(initialText: string = ''): ErrorPanel {
+    const errorPanel = new ErrorPanel();
+    errorPanel.text = initialText;
```

Comment:
Can this be done external to creating the ErrorPanel? That way we can reuse the same code pattern for initialisation as well as updating (L134). Doing so would make it easier to refactor out the common logic for ErrorPanels on initial creation (i.e., L145–154) and updating (L122–134).

Reply:
Yes, we could do that.